public/app/plugins/datasource/jaeger/components/ConfigEditor.tsx

```diff
@@ -20,9 +22,16 @@ export const ConfigEditor: React.FC<Props> = ({ options, onOptionsChange }) => {
       <div className="gf-form-group">
         <TraceToLogsSettings options={options} onOptionsChange={onOptionsChange} />
       </div>
+
       <div className="gf-form-group">
         <NodeGraphSettings options={options} onOptionsChange={onOptionsChange} />
       </div>
+
+      {config.featureToggles.traceToMetrics ? (
```

Comment:
We should prob move this up to below the 'Trace to logs' section so they're together as they provide similar functionality.

Metrics description: 'Trace to metrics lets you link trace spans to Prometheus queries.'
Logs description: 'Trace to logs lets you navigate from a trace span to the selected data source's log.'
^ Can we make these descriptions more alike?

Similarly, the Data source tooltip text is different 'The data source the trace is going to navigate to' & 'The Prometheus data source the trace is going to query'. I think they should have a similar description to create a more pleasant experience.